### PR TITLE
Lk jb jr add review to donut show

### PR DESCRIPTION
**Pull request by LmKupke**

### CODE REVIEW

File: app/controllers/donuts_controller.rb [hunk 1]
```diff
@@ -21,6 +21,10 @@ def create
 
   def show
     @donut = Donut.find(params[:id])
+    @reviews = @donut.reviews
+    @url = "donut_reviews_path"
+    @method = "post"
```
Comment (CeeNoriega) [inline app/controllers/donuts_controller.rb]: not necessary

Comment (CeeNoriega) [inline app/controllers/donuts_controller.rb]: both @url & @method ^^

Comment (CeeNoriega) [inline app/controllers/donuts_controller.rb]: how is the show page using a post method?

File: spec/features/reviews/01_user_adds_a_review_spec.rb [hunk 2]
```diff
@@ -0,0 +1,50 @@
+require "rails_helper"
+
+feature "user reviews a donut", %{
+  As an authenticated user
+  I would like to add a Review to a Donut
+  So that other people know how my donut experience was
+  } do
+  # ACCEPTANCE CRITERIA
+  # Authenticated user sees donut review form on donut show page
+  # Unauthenticated user does not see review form
```
Comment (CeeNoriega) [inline spec/features/reviews/01_user_adds_a_review_spec.rb]: remove this before committing! This is just for your own personal use

File: spec/features/reviews/01_user_adds_a_review_spec.rb [hunk 3]
```diff
@@ -0,0 +1,50 @@
+require "rails_helper"
+
+feature "user reviews a donut", %{
+  As an authenticated user
```
Comment (CeeNoriega) [inline spec/features/reviews/01_user_adds_a_review_spec.rb]: this should be removed

File: app/controllers/donuts_controller.rb (hunk 1 above)
Comment (LmKupke) [inline app/controllers/donuts_controller.rb]: Reviews are nested inside the Donut#show page. Like yelp, someone can add a review at the bottom of the page. Eventually we will have links in line with a users review that allows them to edit the review on the same page

File: spec/features/reviews/01_user_adds_a_review_spec.rb (hunk 2 above)
Comment (LmKupke) [inline spec/features/reviews/01_user_adds_a_review_spec.rb]: Isn't this just personal preference? I've notice Dan P. uses this in his video lectures and commits with them.

**Pull request merged**